app/models/manageiq/providers/amazon/inventory/targets.rb

```diff
@@ -1,4 +1,9 @@
 class ManageIQ::Providers::Amazon::Inventory::Targets < ManageIQ::Providers::Amazon::Inventory
```

Comment:
The target is really an inventory here

We can revisit that Inventory could have target and collector